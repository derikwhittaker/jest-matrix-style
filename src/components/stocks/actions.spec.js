import React, { Component } from 'react';
import { shallow } from 'enzyme';
import sinon from 'sinon';

import StocksActions from './actions';

describe('Stocks.Actions', () => {

    it('Render', () => {
        const wrapper = shallow(<StocksActions />);

        expect(wrapper.html()).toMatchSnapshot();
    });

    it('Snapshot an object graph', () => {
        const someObject = {
            foo: 'bar',
            baz: 'buzz'
        }

        expect(someObject).toMatchSnapshot();
    });

});