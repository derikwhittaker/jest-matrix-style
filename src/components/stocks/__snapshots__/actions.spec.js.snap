// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stocks.Actions Render 1`] = `"<div class=\\"MuiGrid-container-1 stocks-actions-container\\"><div class=\\"MuiGrid-item-2 MuiGrid-grid-xs-12-38 grid-right\\"><button tabindex=\\"0\\" class=\\"MuiButtonBase-root-106 MuiButton-root-91 MuiButton-raised-96 MuiButton-raisedPrimary-97\\" type=\\"button\\"><span class=\\"MuiButton-label-92\\">Purchase</span><span class=\\"MuiTouchRipple-root-109\\"></span></button></div></div>"`;

exports[`Stocks.Actions Snapshot an object graph 1`] = `
Object {
  "baz": "buzz",
  "foo": "bar",
}
`;
